cli/smartcontract/smart_contract.go

```diff
@@ -166,6 +166,10 @@ func NewCommands() []cli.Command {
 						Name:  "no-permissions",
 						Usage: "do not check if invoked contracts are allowed in manifest",
 					},
+					cli.StringFlag{
+						Name:  "bindings",
+						Usage: "file for smart-contract bindings configuration",
```

Comment:
output file? I think we need to be more specific on what's the inputs and what's the outputs.